docs/elements.md

```diff
@@ -1505,17 +1505,17 @@ def generate(data):
 
 Attributes for `<pl-variable-output>`:
 
-| Attribute          | Type    | Default  | Description                                    |
-| ------------------ | ------- | -------- | ---------------------------------------------- |
-| `digits`           | integer | —        | Number of digits to display after the decimal. |
-| `default-tab`      | string  | 'matlab' | Select the active tab.                         |
-| `show-matlab`      | boolean | true     | Toggles the display of the Matlab tab.         |
-| `show-mathematica` | boolean | true     | Toggles the display of the Mathematica tab.    |
-| `show-python`      | boolean | true     | Toggles the display of the Python tab.         |
-| `show-r`           | boolean | true     | Toggles the display of the R tab.              |
-| `show-sympy`       | boolean | true     | Toggles the display of the SymPy tab.          |
+| Attribute          | Type    | Default  | Description                                                              |
+| ------------------ | ------- | -------- | ------------------------------------------------------------------------ |
+| `digits`           | integer | —        | Number of digits to display after the decimal.                           |
+| `default-tab`      | string  | `matlab` | Select the active tab.                                                   |
+| `show-matlab`      | boolean | true     | Toggles the display of the Matlab tab.                                   |
+| `show-mathematica` | boolean | true     | Toggles the display of the Mathematica tab. Also compatible with Octave. |
```

Comment:
Is the comment in the wrong row?

Reply:
Not the first time I've mixed up Matlab and Mathematica 🤦‍♂️

Reply:
Fixed here https://github.com/PrairieLearn/PrairieLearn/pull/7916/commits/ee958df312e9267177bbaa86510c2060a1816cc3